estate/models/estate_property.py

```diff
@@ -0,0 +1,30 @@
+from datetime import timedelta
+from odoo import models, fields
+
+class EstateProperty(models.Model):
+    _name ="estate.property"
+    _description="test description"
+
+
+    name=fields.Char(required=True,default="Unknown")
+    description=fields.Text()
+    postcode=fields.Char()
+    date_availability=fields.Date(copy=False,default=fields.Date.today()+timedelta(days=+90))
```

Comment:
```suggestion
    date_availability = fields.Date(copy=False, default=fields.Date.today() + timedelta(days=90))
```
Please ensure the entire module follows coding guidelines by adding spaces wherever necessary.
Could you explain the purpose of "copy", "required", "readonly" parameters?

Reply:
copy is used to make sure that while duplicating a record the field where copy is used does not gets copied,
required is used to make the field not null, and read only parameter prevents the alteration of the field.